### DISCUSSION TITLE
Hello Team!

### DISCUSSION THREAD

**Discussion by jayananden-m** [OpenMined/courses#67] [Introduce Yourself]
Hello all,

I am Jayananden from India. Currently doing my bachelor's degree. I work on Machine Learning projects. And I am taking this course for getting know about privacy in data which is required for this digizitalized era. 
Hope to learn the need newly!

**Comment by iamtrask** [comment]
Hi Jayananden! Welcome! Thanks for joining the community. I hope you enjoy the course.

**Comment by yemikifouly** [comment]
Welcome Jaya! 🙂 We, too, believe that privacy as it relates to data - and everything else - is very needed in this era. I hope you enjoy the course and learn a lot from it.
